godtools/App/Features/Tools/SwiftUI/ToolCardView/ViewModels/MockToolCardViewModel.swift

```diff
@@ -21,8 +20,8 @@ class MockToolCardViewModel: BaseToolCardViewModel {
             parallelLanguageName = "Arabic (Bahrain) ✓"
         }
         
-        if showBannerImage, let deviceImage = appDiContainer.deviceAttachmentBanners.getDeviceBanner(resourceId: "2") {
-            bannerImage = Image(uiImage: deviceImage)
+        if showBannerImage {
+            bannerImage = Image.from(uiImage: DeviceAttachmentBanners().getDeviceBanner(resourceId: "2"))
```

Comment:
Hey @rachaelblue can you actually inject the DeviceAttachmentBanners into the ViewModel and fetch it from the SwiftUIPreviews AppDiContainer.  I should have been more clear on that.  Eventually the MockDiContainer will provide an instance that can be injected.

Reply:
That makes sense.  Fixed!